src/main.ts

```diff
@@ -289,12 +289,7 @@ export class ConnectionFeature extends SqlOpsFeature<undefined> {
 		};
 
 		let registerOnConnectionChanged = (handler: (changedConnInfo: azdata.ChangedConnectionInfo) => any): void => {
-			client.onNotification(protocol.ConnectionChangedNotification.type, (params: protocol.ConnectionChangedParams) => {
-				handler({
```

Comment:
This is the only actual "change" in this PR - if for some reason the object we got passed had additional properties on it then we would now be passing those along to the provider. But this isn't how the rest of the handlers are set up anyways - and since this is coming from the server side we have control over that for MSSQL (and other providers shouldn't be sending additional data anyways).